app/controllers/order_item_controller.rb

```diff
@@ -0,0 +1,39 @@
+
+class OrderItemController < ApplicationController
```

Comment:
This should be order_items_controller/OrderItemsController, controllers should be plural while models are singular